### DISCUSSION TITLE
Meu projeto foi considerado pelo GitHub um dos top open source projects de 2018!

### DISCUSSION THREAD

**Discussion by leonardomso** [frontendbr/forum#1319] [Dicas]
Esse ano eu desenvolvi um projeto open source chamado **[33 concepts every JavaScript developer should know](https://github.com/leonardomso/33-js-concepts)** baseado em um [artigo](https://medium.com/@stephenthecurt/33-fundamentals-every-javascript-developer-should-know-13dd720a90d1) escrito por um desenvolvedor chamado Stephen Curtis. 

Basicamente, esse projeto faz uma listagem de 33 conceitos de JavaScript que são importantes de se ter conhecimento e que são muito usados no dia-a-dia, o projeto tem o intuito de ajudar os mais iniciantes a começarem com o JavaScript e ser ao menos um guia para que eles possam consultar alguns conceitos que eles ainda não sabem e possam ler mais sobre.

Recentemente, meu projeto foi **considerado pelo GitHub** como um dos **[top open source projects de 2018!](https://blog.github.com/2018-12-13-new-open-source-projects/)**

Estou criando essa issue só pra agradecer a uma galera da comunidade BR que de um feedback positivo enorme, e também aproveitar pra divulgar pra quem ainda não conhece que vá lá e dê uma conferida. Obrigado a quem contribuiu, ajudou, deu feedback, e para aqueles que quiserem contribuir, por favor sintam-se livre também!

👉 **[33 concepts every JavaScript developer should know](https://github.com/leonardomso/33-js-concepts)**

🚀🎉 **[Top Open Source Projets of 2018](https://blog.github.com/2018-12-13-new-open-source-projects/)**

**Comment by expalmer** [comment]
Meus parabéns pela iniciativa e a contribuição.

**Comment by felipefialho** [comment]
Sensacional 👏 👏 👏 👏

**Comment by nonatodiego** [comment]
Parabéns pela conquista ! Eu como novato em dev. fui muito beneficiado por esse projeto, estou aprendendo bastante. Valeu ! 👏 👏

**Comment by hevertoncastro** [comment]
Parabéns @leonardomso 

Excelente repositório, muito conteúdo útil!

PS: Você é de Franca cara!? Precisamos tomar um café e falar de javascript!

**Comment by leonardomso** [comment]
@hevertoncastro você é daqui também? Me chama no Twitter que a gente marca sim!!

**Comment by fabiorocha** [comment]
Fenomenal! Parabéns @leonardomso! 🎉🎉🎉🎉

**Comment by fabiochiquezi** [comment]
Parabénssssss!!!!
